splink/comparison_template_library.py

```diff
@@ -1318,3 +1316,333 @@ def __init__(
             "comparison_levels": comparison_levels,
         }
         super().__init__(comparison_dict)
+
+
+class EmailComparisonBase(Comparison):
+    def __init__(
+        self,
+        col_name: str,
+        invalid_emails_as_null: bool = False,
+        valid_email_regex: str = "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+[.][a-zA-Z]{2,}$",
+        term_frequency_adjustments_full: bool = False,
+        levenshtein_thresholds: int | list = [],
+        damerau_levenshtein_thresholds: int | list = [],
+        jaro_winkler_thresholds: float | list = [0.88],
```

Comment:
can we also add a level for jaro below JW - I think there are cases where is could be useful